src/pages/items/ItemsPage.tsx

```diff
@@ -38,6 +39,7 @@ interface List {
 function ItemsPage() {
     // Local state
     const [lists, setLists] = useRecoilState(itemsState)
```

Comment:
I just realized i named this variable wrong :D 
Can you just rename it in this PR, I think this is meant to be items/categories

Reply:
Naming is hard :D. Even more when you work with someone ;)